src/renderer/entities/transaction/ui/QrCode/QrReader/SignatureReaderError.tsx

```diff
@@ -0,0 +1,32 @@
+import { Button, FootnoteText } from '@shared/ui';
+import { useI18n } from '@app/providers';
+import { CameraError, CameraErrorText } from '../common/constants';
+
+type Props = {
+  error?: CameraError;
+  onTryAgain: () => void;
+  isCameraOn: boolean;
+};
+
+export const SignatureReaderError = ({ error, onTryAgain, isCameraOn }: Props) => {
+  const { t } = useI18n();
+  const showTryAgainButton =
+    error && [CameraError.UNKNOWN_ERROR, CameraError.DENY_ERROR, CameraError.DECODE_ERROR].includes(error);
```

Comment:
Maybe `[CameraError.UNKNOWN_ERROR, CameraError.DENY_ERROR, CameraError.DECODE_ERROR].includes(error)` to variable? 

`showTryAgainButton` sounds like function name, maybe rename it?